src/script/standard.cpp

```diff
@@ -274,14 +274,11 @@ class CScriptVisitor : public boost::static_visitor<CScript>
         return CScript() << CScript::EncodeOP_N(id.version) << std::vector<unsigned char>(id.program, id.program + id.length);
     }
 };
-
-const CScriptVisitor g_script_visitor;
-
 } // namespace
 
 CScript GetScriptForDestination(const CTxDestination& dest)
 {
-    return boost::apply_visitor(::g_script_visitor, dest);
+    return boost::apply_visitor(CScriptVisitor{}, dest);
```

Comment:
I think this is an improvement in any case. No value in using a global here, the only case where a setup like this would make sense is when there is an expensive pre-computation.

Reply:
This is also restoring the way it has always been before commit 5f72ddb